ui/app/serializers/config-ui/message.js

```diff
@@ -8,10 +8,32 @@ import ApplicationSerializer from '../application';
 export default class MessageSerializer extends ApplicationSerializer {
   primaryKey = 'id';
 
+  serialize() {
+    const json = super.serialize(...arguments);
+    json.start_time = new Date(json.start_time).toISOString();
+    json.end_time = new Date(json.end_time).toISOString();
```

Comment:
I don't think this is necessary if you use the [built in date transform](https://api.emberjs.com/ember-data/release/classes/datetransform/).

Reply:
I ran into some issues here using date transform since the datetime-local picker expects the values to be in a string format 🫠 @zofskeez

Reply:
Update: I was able to get the date transform working!